src/graph/graph.h

```diff
@@ -30,28 +30,18 @@ typedef struct {
     DataBlock *edges;               // Graph edges stored in blocks.
     Edge *_edgesHashTbl;            // Hash table containing edges.
     GrB_Matrix adjacency_matrix;    // Adjacency matrix, holds all graph connections.
-    GrB_Matrix *_relations;         // Relation matrices.
-    size_t relation_cap;            // Number of relations graph can hold.
+    GrB_Matrix *relations;         // Relation matrices.
     size_t relation_count;          // Number of relation matrices.
-    GrB_Matrix *_labels;            // Label matrices.
-    size_t label_cap;               // Number of labels graph can hold.
+    GrB_Matrix *labels;            // Label matrices.
```

Comment:
Renamed this and `_labels` because the GraphContext reallocs store arrays and matrix arrays at the same time - don't want to duplicate effort.